lib/install/install.sh

```diff
@@ -155,16 +155,72 @@ bpkg_install () {
 
   echo
 
+  local build_output=""
+  local build=""
+  local json=""
+
   for pkg in "${pkgs[@]}"; do
     if test -d "$(bpkg_realpath "$pkg" 2>/dev/null)"; then
-      if ! (cd "$pkg" && ${BPKG_DEPS_EXEC}); then
-        return 1
-      fi
+      pushd . >/dev/null || return 1
+      cd "$pkg" || return 1
+
+      pwd
+      ${BPKG_DEPS_EXEC} || return 1
 
       did_fail=0
+
+      ## handle global install on a given local directory path
+      if (( needs_global )); then
+        if test -f "bpkg.json"; then
+          json="$(cat bpkg.json 2>/dev/null)"
+        elif test -f "package.json"; then
+          json="$(cat package.json 2>/dev/null)"
+        else
+          popd >/dev/null || return 1
+          bpkg_warn 'Unable to determine bpkg.json'
```

Comment:
I think this `else` block could be removed, since this error case is covered by the `[ -z "$json" ]` test below?